docs-2.0/3.ngql-guide/6.funtions-and-expressions/3.date-and-time.md

```diff
@@ -8,3 +8,5 @@ int now()  |Return the current date and time. |
 date date() |Return the current date based on the current system.
 time time() |Return the current calendar time of the current time zone.
 datetime datetime() |Return the current datetime based on the current time.
+
+The date(), time(), and datetime() functions accept three kind of parameters, namely empty, string, and map.
```

Comment:
I don't understand:
 how a map can be a parameter of date() ?

Reply:
Suck as `  date({year:1984, month:11, day:11})`

Reply:
awesome